torch/csrc/autograd/profiler_legacy.cpp

```diff
@@ -368,6 +368,24 @@ std::vector<std::vector<int64_t>> inputSizes(const at::RecordFunction& fn) {
   return sizes;
 }
 
+std::vector<std::string> inputTypes(const at::RecordFunction& fn) {
```

Comment:
could you move this function to profiler_kineto.h/cpp ?

Reply:
will do

Reply:
> Thank you, looks good, modulo one minor comment. Also there might be some (trivial) merge conflict, so please rebase and we'll continue with landing.

merged the latest code in master branch